src/auth/index.js

```diff
@@ -118,6 +118,41 @@ export default class Auth {
       .then(responseHandler);
   }
 
+  /**
+   * Exchanges a code obtained via a native social authentication solution for the user's tokens
+   *
+   * @param {Object} parameters parameters used to obtain tokens from a code
+   * @param {String} parameters.subjectToken code returned by the native social authentication solution
+   * @param {String} parameters.subjectTokenType identifier that indicates the type of `subjectToken`
+   * @param {String} parameters.userProfile optional element used for native iOS interactions for which profile updates can occur. Only for `APPLE-AUTHZ-CODE`
+   * @param {String} parameters.audience optional API audience to request
+   * @param {String} parameters.scope optional scopes to request
+   * @returns {Promise}
+   *
```

Comment:
add

```
@see https://auth0.com/docs/api/authentication#token-exchange-for-native-social
```

Reply:
Added in 0758eb8fbb8a59f43b77780b9bd125f46da6bf35. Thanks, great review BTW